docker-compose.yml

```diff
@@ -35,6 +35,8 @@ services:
       LOCAL_HOST: 0.0.0.0:3000
       PRODUCTION_HOST: sleepy-brushlands-18624.herokuapp.com
       REDIS_URL: redis://redis
+      AWS_ACCESS_KEY_ID: AKIA3C4UHVZWMIRL6BPF
```

Comment:
why do you need this sensitive information in public?

Reply:
Thanks. I hide it in config/secrets.yml.